src/controllers/session/login/index.html

```diff
@@ -9,7 +9,10 @@ <h1 class="sectionTitle">${HeaderPleaseSignIn}</h1>
             </div>
 
             <div class="inputContainer">
-                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password" />
+                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password">
+                <button is="paper-icon-button-light" class="passwordToggle" type="button">
+                    <span class="material-icons visibility"></span>
```

Comment:
All icons must have `aria-hidden="true"` so that they are not processed by screen readers.
```suggestion
                    <span class="material-icons visibility" aria-hidden="true"></span>
```